virt/virt-2-6-release-notes.adoc

```diff
@@ -32,10 +32,26 @@ Other operating system templates shipped with {VirtProductName} are not supporte
 
 //CNV-7544: Microsoft's Windows Server Virtualization Validation Program (SVVP) applies to RHCOS workers
 
+* OpenShift Virtualization is certified in Microsoft’s Windows Server Virtualization Validation Program (SVVP) to run Windows Server workloads.
++
+The SVVP Certification applies to:
++
+** Red Hat Enterprise Linux CoreOS workers. In the Microsoft SVVP Catalog, they are named Red Hat OpenShift Container Platform 4 on RHEL CoreOS.
+
+** Intel and AMD CPUs.
+
 //CNV-9723: VMs are now migrated when a node is drained
 
 //CNV-9521: VMs that use UEFI can now be booted by using secure boot
 
+* You can boot a virtual machine (VM) in Extensible Firmware Interface (EFI) mode.
```

Comment:
Can you add an xref to your documentation for this feature? Also, add the word "now" as in "You can now boot..."